Crash when writing content stats and intermediate output directory does not exist

<!--
# Please make sure that the issue is present in the
# develop branch of MonoGame before reporting
#
# You can download the development build installer from:
# http://www.monogame.net/downloads/
-->

<!-- Write your issue below -->
I cannot build any project using the Xml Importer on MacOs, using the latest Monogame & Content Pipeline version on TeamCity (3.8.0.76). I attached a minimal project that leads to the error.

[TestXml.zip](https://github.com/MonoGame/MonoGame/files/2476375/TestXml.zip)

Here is the error:
```
Projet "/Users/picnic/Projects/TestXml/TestXml/TestXml.MacOS.csproj" (Build cibles) :

RunContentBuilder cible :
    /Library/Frameworks/Mono.framework/Versions/Current/bin/mono "/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/Tools/MGCB.exe" /quiet /platform:DesktopGL /@:"/Users/picnic/Projects/TestXml/TestXml/Content/Content.mgcb" /outputDir:"bin/DesktopGL/Content" /intermediateDir:"obj/DesktopGL/Content"
    /Users/picnic/Projects/TestXml/TestXml/Content/Example.xml
    EXEC : warning : Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
    /Users/picnic/Projects/TestXml/TestXml/Content/Example.xml : error : Importer 'XmlImporter' had unexpected failure!
    
    Unhandled Exception:
    System.IO.DirectoryNotFoundException: Could not find a part of the path "/Users/picnic/Projects/TestXml/TestXml/Content/obj/DesktopGL/Content/.mgstats".
      at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x00164] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options, System.String msgPath, System.Boolean bFromProxy, System.Boolean useLongPath, System.Boolean checkHost) [0x00000] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions,string,bool,bool,bool)
      at System.IO.StreamWriter.CreateFile (System.String path, System.Boolean append, System.Boolean checkHost) [0x0001c] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize, System.Boolean checkHost) [0x00055] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize) [0x00000] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding) [0x00000] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at (wrapper remoting-invoke-with-check) System.IO.StreamWriter..ctor(string,bool,System.Text.Encoding)
      at Microsoft.Xna.Framework.Content.Pipeline.ContentStatsCollection.Write (System.String outputPath) [0x00012] in <227b2d909358404f981099d80e2439b8>:0 
      at MGCB.BuildContent.Build (System.Int32& successCount, System.Int32& errorCount) [0x006df] in <764b3cbebab042cf8276bccb48963fef>:0 
      at MGCB.Program.Main (System.String[] args) [0x00089] in <764b3cbebab042cf8276bccb48963fef>:0 
    [ERROR] FATAL UNHANDLED EXCEPTION: System.IO.DirectoryNotFoundException: Could not find a part of the path "/Users/picnic/Projects/TestXml/TestXml/Content/obj/DesktopGL/Content/.mgstats".
      at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x00164] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options, System.String msgPath, System.Boolean bFromProxy, System.Boolean useLongPath, System.Boolean checkHost) [0x00000] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions,string,bool,bool,bool)
      at System.IO.StreamWriter.CreateFile (System.String path, System.Boolean append, System.Boolean checkHost) [0x0001c] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize, System.Boolean checkHost) [0x00055] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize) [0x00000] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding) [0x00000] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at (wrapper remoting-invoke-with-check) System.IO.StreamWriter..ctor(string,bool,System.Text.Encoding)
      at Microsoft.Xna.Framework.Content.Pipeline.ContentStatsCollection.Write (System.String outputPath) [0x00012] in <227b2d909358404f981099d80e2439b8>:0 
      at MGCB.BuildContent.Build (System.Int32& successCount, System.Int32& errorCount) [0x006df] in <764b3cbebab042cf8276bccb48963fef>:0 
      at MGCB.Program.Main (System.String[] args) [0x00089] in <764b3cbebab042cf8276bccb48963fef>:0 
    System.Reflection.ReflectionTypeLoadException: Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
      at (wrapper managed-to-native) System.Reflection.Assembly.GetTypes(System.Reflection.Assembly,bool)
      at System.Reflection.Assembly.GetTypes () [0x00000] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateSerializer+<>c.<FindType>b__15_0 (System.Reflection.Assembly assembly) [0x00000] in <227b2d909358404f981099d80e2439b8>:0 
      at System.Linq.Enumerable+<SelectManyIterator>d__167`3[TSource,TCollection,TResult].MoveNext () [0x0004e] in <cc3b329d40bd4675ae0e985e302972af>:0 
      at System.Linq.Enumerable+WhereSelectEnumerableIterator`2[TSource,TResult].MoveNext () [0x00059] in <cc3b329d40bd4675ae0e985e302972af>:0 
      at System.Linq.Enumerable.TryGetFirst[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Boolean& found) [0x00045] in <cc3b329d40bd4675ae0e985e302972af>:0 
      at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00000] in <cc3b329d40bd4675ae0e985e302972af>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateSerializer.FindType (System.String typeName) [0x00202] in <227b2d909358404f981099d80e2439b8>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateReader.ReadTypeName () [0x00012] in <227b2d909358404f981099d80e2439b8>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateReader.ReadObject[T] (Microsoft.Xna.Framework.Content.ContentSerializerAttribute format, Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.ContentTypeSerializer typeSerializer, T existingInstance) [0x0009f] in <227b2d909358404f981099d80e2439b8>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateReader.ReadObject[T] (Microsoft.Xna.Framework.Content.ContentSerializerAttribute format) [0x00017] in <227b2d909358404f981099d80e2439b8>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateSerializer.Deserialize[T] (System.Xml.XmlReader input, System.String referenceRelocationPath) [0x00053] in <227b2d909358404f981099d80e2439b8>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.XmlImporter.Import (System.String filename, Microsoft.Xna.Framework.Content.Pipeline.ContentImporterContext context) [0x00007] in <227b2d909358404f981099d80e2439b8>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.ContentImporter`1[T].Microsoft.Xna.Framework.Content.Pipeline.IContentImporter.Import (System.String filename, Microsoft.Xna.Framework.Content.Pipeline.ContentImporterContext context) [0x0001c] in <227b2d909358404f981099d80e2439b8>:0 
      at MonoGame.Framework.Content.Pipeline.Builder.PipelineManager.ProcessContent (MonoGame.Framework.Content.Pipeline.Builder.PipelineBuildEvent pipelineEvent) [0x00079] in <227b2d909358404f981099d80e2439b8>:0 
    /Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/MonoGame.Content.Builder.targets(84,5): error MSB3073: La commande "/Library/Frameworks/Mono.framework/Versions/Current/bin/mono "/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/Tools/MGCB.exe" /quiet /platform:DesktopGL /@:"/Users/picnic/Projects/TestXml/TestXml/Content/Content.mgcb" /outputDir:"bin/DesktopGL/Content" /intermediateDir:"obj/DesktopGL/Content"" s'est arrêtée avec le code 1.
Génération de la cible "RunContentBuilder" terminée dans le projet "TestXml.MacOS.csproj" -- ÉCHEC.

Génération du projet "TestXml.MacOS.csproj" terminée -- ÉCHEC.

ÉCHEC de la build.

/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/msbuild/15.0/bin/Microsoft.Common.CurrentVersion.targets(2126,5): warning MSB3245: Impossible de résoudre cette référence. Impossible de trouver l'assembly "MonoGame.Framework". Vérifiez que l'assembly existe sur le disque. Si elle est requise par votre code, vous risquez d'obtenir des erreurs de compilation.
/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/msbuild/15.0/bin/Microsoft.Common.CurrentVersion.targets(2126,5): warning MSB3243: Impossible de résoudre le conflit entre "MonoGame.Framework, Version=3.8.0.76, Culture=neutral, PublicKeyToken=null" et "MonoGame.Framework". Choix arbitraire de "MonoGame.Framework, Version=3.8.0.76, Culture=neutral, PublicKeyToken=null".
EXEC : warning : Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
/Users/picnic/Projects/TestXml/TestXml/Content/Example.xml : error : Importer 'XmlImporter' had unexpected failure!
/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/MonoGame.Content.Builder.targets(84,5): error MSB3073: La commande "/Library/Frameworks/Mono.framework/Versions/Current/bin/mono "/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/Tools/MGCB.exe" /quiet /platform:DesktopGL /@:"/Users/picnic/Projects/TestXml/TestXml/Content/Content.mgcb" /outputDir:"bin/DesktopGL/Content" /intermediateDir:"obj/DesktopGL/Content"" s'est arrêtée avec le code 1.
```

<!-- System stats -->

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8.0.76

#### What operating system are you using:
- MacOs

#### What MonoGame platform are you using:
<!-- e.g. DesktopGL, WindowsDX, WindowsUWP, Android -->
- DesktopGL

The package has been built on TeamCity so I was able to test your fix. It fixed the first exception:
(System.IO.DirectoryNotFoundException: Could not find a part of the path "/Users/picnic/Projects/TestXml/TestXml/Content/obj/DesktopGL/Content/.mgstats")

However, the second exception regarding the XML file loading remains. Here is the message I get when I try to compile the project with the last build of monogame + pipeline:
```
Projet "/Users/picnic/Projects/TestXml/XmlGameData/XmlGameData.csproj" (Build cibles) :

ResolveAssemblyReferences cible :
    Une liste d'exclusion de profil TargetFramework sera générée.
GenerateTargetFrameworkMonikerAttribute cible :
  La cible est ignorée "GenerateTargetFrameworkMonikerAttribute", car tous les fichiers de sortie sont à jour par rapport aux fichiers d'entrée.
CoreCompile cible :
    /Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/msbuild/15.0/bin/Roslyn/csc.exe /noconfig /nowarn:1701,1702,2008 /nostdlib+ /errorreport:prompt /warn:4 /define:DEBUG /errorendlocation /preferreduilang:fr-FR /reference:/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/Assemblies/DesktopGL/MonoGame.Framework.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/mscorlib.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/mscorlib.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/Microsoft.CSharp.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.Core.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.Net.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.Runtime.Serialization.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.ServiceModel.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.ServiceModel.Web.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.Windows.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.Xml.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.Xml.Linq.dll /reference:/Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild-frameworks/.NETPortable/v4.0/Profile/Profile158/System.Xml.Serialization.dll /debug+ /debug:portable /optimize- /out:obj/Debug/XmlGameData.dll /target:library /utf8output Properties/AssemblyInfo.cs XmlData.cs "/var/folders/2c/scpghqvj5tjfz768r39m2d6c0000gn/T/.NETPortable,Version=v4.0,Profile=Profile158.AssemblyAttributes.cs"
_CopyFilesMarkedCopyLocal cible :
    Mise à jour de l'horodatage "/Users/picnic/Projects/TestXml/XmlGameData/obj/Debug/XmlGameData.csproj.CopyComplete".
CopyFilesToOutputDirectory cible :
    Copie du fichier de "obj/Debug/XmlGameData.dll" vers "bin/Debug/XmlGameData.dll".
    XmlGameData -> /Users/picnic/Projects/TestXml/XmlGameData/bin/Debug/XmlGameData.dll
    Copie du fichier de "obj/Debug/XmlGameData.pdb" vers "bin/Debug/XmlGameData.pdb".
__________________________________________________
Projet "/Users/picnic/Projects/TestXml/TestXml/TestXml.MacOS.csproj" (Build cibles) :

RunContentBuilder cible :
    /Library/Frameworks/Mono.framework/Versions/Current/bin/mono "/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/Tools/MGCB.exe" /quiet /platform:DesktopGL /@:"/Users/picnic/Projects/TestXml/TestXml/Content/Content.mgcb" /outputDir:"bin/DesktopGL/Content" /intermediateDir:"obj/DesktopGL/Content"
    /Users/picnic/Projects/TestXml/TestXml/Content/Example.xml
    EXEC : warning : Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
    /Users/picnic/Projects/TestXml/TestXml/Content/Example.xml : error : Importer 'XmlImporter' had unexpected failure!
    System.Reflection.ReflectionTypeLoadException: Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
      at (wrapper managed-to-native) System.Reflection.Assembly.GetTypes(System.Reflection.Assembly,bool)
      at System.Reflection.Assembly.GetTypes () [0x00000] in <71f4e3d7d65e42b0a699d02e59766d3c>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateSerializer+<>c.<FindType>b__15_0 (System.Reflection.Assembly assembly) [0x00000] in <ee3504a62da744b2be82d562251054fd>:0 
      at System.Linq.Enumerable+<SelectManyIterator>d__167`3[TSource,TCollection,TResult].MoveNext () [0x0004e] in <cc3b329d40bd4675ae0e985e302972af>:0 
      at System.Linq.Enumerable+WhereSelectEnumerableIterator`2[TSource,TResult].MoveNext () [0x00059] in <cc3b329d40bd4675ae0e985e302972af>:0 
      at System.Linq.Enumerable.TryGetFirst[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Boolean& found) [0x00045] in <cc3b329d40bd4675ae0e985e302972af>:0 
      at System.Linq.Enumerable.FirstOrDefault[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00000] in <cc3b329d40bd4675ae0e985e302972af>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateSerializer.FindType (System.String typeName) [0x00202] in <ee3504a62da744b2be82d562251054fd>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateReader.ReadTypeName () [0x00012] in <ee3504a62da744b2be82d562251054fd>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateReader.ReadObject[T] (Microsoft.Xna.Framework.Content.ContentSerializerAttribute format, Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.ContentTypeSerializer typeSerializer, T existingInstance) [0x0009f] in <ee3504a62da744b2be82d562251054fd>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateReader.ReadObject[T] (Microsoft.Xna.Framework.Content.ContentSerializerAttribute format) [0x00017] in <ee3504a62da744b2be82d562251054fd>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate.IntermediateSerializer.Deserialize[T] (System.Xml.XmlReader input, System.String referenceRelocationPath) [0x00053] in <ee3504a62da744b2be82d562251054fd>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.XmlImporter.Import (System.String filename, Microsoft.Xna.Framework.Content.Pipeline.ContentImporterContext context) [0x00007] in <ee3504a62da744b2be82d562251054fd>:0 
      at Microsoft.Xna.Framework.Content.Pipeline.ContentImporter`1[T].Microsoft.Xna.Framework.Content.Pipeline.IContentImporter.Import (System.String filename, Microsoft.Xna.Framework.Content.Pipeline.ContentImporterContext context) [0x0001c] in <ee3504a62da744b2be82d562251054fd>:0 
      at MonoGame.Framework.Content.Pipeline.Builder.PipelineManager.ProcessContent (MonoGame.Framework.Content.Pipeline.Builder.PipelineBuildEvent pipelineEvent) [0x00079] in <ee3504a62da744b2be82d562251054fd>:0 
    /Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/MonoGame.Content.Builder.targets(84,5): error MSB3073: La commande "/Library/Frameworks/Mono.framework/Versions/Current/bin/mono "/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/Tools/MGCB.exe" /quiet /platform:DesktopGL /@:"/Users/picnic/Projects/TestXml/TestXml/Content/Content.mgcb" /outputDir:"bin/DesktopGL/Content" /intermediateDir:"obj/DesktopGL/Content"" s'est arrêtée avec le code 1.
Génération de la cible "RunContentBuilder" terminée dans le projet "TestXml.MacOS.csproj" -- ÉCHEC.

Génération du projet "TestXml.MacOS.csproj" terminée -- ÉCHEC.

ÉCHEC de la build.

EXEC : warning : Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
/Users/picnic/Projects/TestXml/TestXml/Content/Example.xml : error : Importer 'XmlImporter' had unexpected failure!
/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/MonoGame.Content.Builder.targets(84,5): error MSB3073: La commande "/Library/Frameworks/Mono.framework/Versions/Current/bin/mono "/Library/Frameworks/Mono.framework/External/xbuild/MonoGame/v3.0/Tools/MGCB.exe" /quiet /platform:DesktopGL /@:"/Users/picnic/Projects/TestXml/TestXml/Content/Content.mgcb" /outputDir:"bin/DesktopGL/Content" /intermediateDir:"obj/DesktopGL/Content"" s'est arrêtée avec le code 1.
    1 Avertissement(s)
    2 Erreur(s)
```

--- a/MonoGame.Framework.Content.Pipeline/ContentStatsCollection.cs
+++ b/MonoGame.Framework.Content.Pipeline/ContentStatsCollection.cs
@@ -196,6 +196,8 @@ namespace Microsoft.Xna.Framework.Content.Pipeline
         /// <param name="outputPath">The folder to write the .mgstats file.</param>
         public void Write(string outputPath)
         {
+            // ensure the output folder exists
+            Directory.CreateDirectory(outputPath);
             var filePath = Path.Combine(outputPath, Extension);
             using (var textWriter = new StreamWriter(filePath, false, new UTF8Encoding(false)))
             {

